api/routes_v1.php

```diff
@@ -11,6 +11,8 @@
     new Api\Route(['get'], '/template/{templateId}', Api\Template\GetTemplate::class),
     new Api\Route(['get'], '/template/{docType}/{templateKey}', Api\Template\GetTemplate::class),
     new Api\Route(['get'], '/template/{docType}/{templateKey}/history', Api\Template\ListVersions::class),
+  # Partial
+    new Api\Route(['post'], '/partial', Api\Partial\AddPartial::class),
```

Comment:
Where is the route to get a partial?  

Also, I need to specify whether I want to get pure partials, or partial images.  See previous comment regarding our discussion.

Reply:
The GET endpoints are actually in the next sprint because this task was a bit bigger than just adding a POST endpoint.

https://homeceu.atlassian.net/browse/CEMS-2308